hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1863,11 +1863,12 @@
             Requires `EnMasking` parameter to be enabled.
             SW randomly configures the KMAC in any hashing mode/strength, and enable EDN mode.
             Randomly enable/disable the `entropy_refresh`.
-            // TODO - error handling is not complete, do not enable `wait_timer` yet.
-            KMAC should send request to EDN once `CmdStart` is written, and should send out another
-            request to EDN when either:
+            Randomly configure `wait_timer` values (zero for disable, non-zero for timer expire).
+            The `wait_timer` should be greater than the expected EDN latency.
+            KMAC should send EDN request after `entropy_ready` is set.
+            KMAC also should send out another request to EDN when either:
               - kmac hash counter hits the configured threshold (assuming it is non-zero)
-              - a hash operation is completed, KMAC will refresh its internal entropy state
+              - Hash count exceeds the threshold.
```

Comment:
Oh also @johannheyszl pointed out from https://github.com/lowRISC/opentitan/issues/14123. I think this test should be V2. I think we wrote as V3 because the entropy feature was not finalized at that time?

Reply:
I will update the PR to move this to V2.

Reply:
tyty